### ISSUE TITLE
StudioEventEmitter2D doesn't play sound

### ISSUE THREAD

**Issue by stephanbogner** [alessandrofama/fmod-for-godot#32]
### FMOD for Godot version

plugin.cfg says 1.0 ... but I am pretty sure it's FMOD 2.02.17 for Godot 4.1.1 stable v1.0-alpha-4

### System information

macOS 14.1.2 with Godot 4.1.1.

### Issue description

**First of all:**
Thanks for creating this plugin! Sad that FMOD has no official support for Godot yet so this fills an important void.

### Issue
- When I add a "StudioEventEmitter2D" it doesn't seem to work when played in the game
- I tried "play_event" "ready" and "enter_tree" as well as triggering it via GDScript but nothing made a sound play
- I connected it correctly because when I play it from the plugin's "FMOD Project Browser" ([after I switched to a bigger monitor](https://github.com/alessandrofama/fmod-for-godot/issues/25)), it works as expected

### Errors in debugger:
```
W 0:00:00:0372   push_warning: [WARNING]: The requested event, parameter, bus or vca could not be found. get_event_by_id in src/api/studio_api.cpp:148
  <C++ Source>   ./core/variant/variant_utility.cpp:925 @ push_warning()

W 0:00:00:0372   push_warning: [WARNING]: An invalid object handle was used. load_sample_data in src/api/studio_api.cpp:1117
  <C++ Source>   ./core/variant/variant_utility.cpp:925 @ push_warning()

W 0:00:00:0372   push_warning: [WARNING]: An invalid object handle was used. create_instance in src/api/studio_api.cpp:1070
  <C++ Source>   ./core/variant/variant_utility.cpp:925 @ push_warning()

W 0:00:00:0372   push_warning: [WARNING]: An invalid object handle was used. start in src/api/studio_api.cpp:1449
  <C++ Source>   ./core/variant/variant_utility.cpp:925 @ push_warning()
```

### Steps to reproduce

1. Setup event in FMOD
2. Add StudioEventEmitter2D in Godot project
3. Select "Play event" "ready" or "enter_tree"
4. Connect to event
5. Run scene

### Minimal reproduction project

(I removed the addon for obvious size reasons)
[sound-pitch-test-bug.zip](https://github.com/alessandrofama/fmod-for-godot/files/14121783/sound-pitch-test-bug.zip)

**Comment by alessandrofama** [comment]
I can’t look at your project till next week but your errors usually indicate that you haven’t loaded the banks. Make sure you have loaded the Master, Master.strings banks. If your event is inside any other bank, make sure to load that bank too before trying to play the event.

**Comment by stephanbogner** [comment]
**Thanks for the hint.**

**Try 1:**
I now created a `StudioBankLoader`, set it to load on `ready` and selected my master bank. `StudioEventEmitter2D` is set to `enter tree` but it still didn't play anything. I assume because the bank takes a while to load?

**Try 2:**
I then added the bank to "Banks to Load at Startup" and now StudioEventEmitter2D works 🎉.

**Context:**
My confusion came because I went to [the tutorial page](https://alessandrofama.com/tutorials/fmod/godot/) and then thought:
"_'Playing FMOD Events in Godot 4'_ that's what I need!" clicked on it and since it didn't say "Loading FMOD Banks in Godot 4" being a prerequisite I didn't know I was missing something.

I saw issue #29 which is a great idea! Then we could contribute on aspects like this :)
